plenum/common/tracker.py

```diff
@@ -0,0 +1,33 @@
+from stp_core.common.log import Logger
+
+logger = Logger()
+
+
+class Tracker:
+
+    def __init__(self):
+        self.un_committed = []
+
+    def track(self, state_root, ledger_size):
+
+        un_committed_state = ()
+
+        if state_root != "":
+            un_committed_state += (state_root,)
+        else:
+            raise logger.error("Empty state root given")
```

Comment:
You are raising a logger output (which is not an exception).
You can use `raise LogicError` or `raise PlenumValueError` instead.